README.md

```diff
@@ -62,6 +61,11 @@ docker version
 $ brew install git make jq qemu
 ```
 
+#### On Apple Silicon (ARM64) 
```

Comment:
Looks like you also need to take care of Yetus complaints @vmlemon

Reply:
Hopefully, that should make Yetus yeet it, this time...